contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -0,0 +1,959 @@
+            if (_wethPositionBalance + _oethbPositionBalance > 0) {
+                positionManager.approve(address(clGauge), tokenId);
+                clGauge.deposit(tokenId);
+                lpTokenDepositedToGauge = true;
+            }
+        }
+    }
+
+    /// @notice the constructor
+    /// @param _stratConfig the basic strategy configuration
+    /// @param _wethAddress Address of the Erc20 WETH Token contract
+    /// @param _oethbAddress Address of the Erc20 OETHb Token contract
+    /// @param _swapRouter Address of the Aerodrome Universal Swap Router
+    /// @param _nonfungiblePositionManager Address of position manager to add/remove
+    ///         the liquidity
+    /// @param _clPool Address of the Aerodrome concentrated liquidity pool
+    /// @param _clGauge Address of the Aerodrome slipstream pool gauge
+    /// @param _sugarHelper Address of the Aerodrome Sugar helper contract
+    /// @param _lowerBoundingTick Smaller bounding tick of our liquidity position
+    /// @param _upperBoundingTick Larger bounding tick of our liquidity position
+    /// @param _tickClosestToParity Tick that is closer to 1:1 price parity
+    constructor(
+        BaseStrategyConfig memory _stratConfig,
+        address _wethAddress,
+        address _oethbAddress,
+        address _swapRouter,
+        address _nonfungiblePositionManager,
+        address _clPool,
+        address _clGauge,
+        address _sugarHelper,
+        int24 _lowerBoundingTick,
+        int24 _upperBoundingTick,
+        int24 _tickClosestToParity
+    ) InitializableAbstractStrategy(_stratConfig) {
+        WETH = _wethAddress;
+        OETHb = _oethbAddress;
+        swapRouter = ISwapRouter(_swapRouter);
+        positionManager = INonfungiblePositionManager(
+            _nonfungiblePositionManager
+        );
+        clPool = ICLPool(_clPool);
+        clGauge = ICLGauge(_clGauge);
+        helper = ISugarHelper(_sugarHelper);
+        sqrtRatioX96TickLower = ISugarHelper(_sugarHelper).getSqrtRatioAtTick(
+            _lowerBoundingTick
+        );
+        sqrtRatioX96TickHigher = ISugarHelper(_sugarHelper).getSqrtRatioAtTick(
+            _upperBoundingTick
+        );
+        sqrtRatioX96TickClosestToParity = ISugarHelper(_sugarHelper)
+            .getSqrtRatioAtTick(_tickClosestToParity);
+
+        lowerTick = _lowerBoundingTick;
+        upperTick = _upperBoundingTick;
+        tickSpacing = 1;
+
+        require(
+            ICLPool(_clPool).token0() == _wethAddress,
+            "Only WETH supported as token0"
+        );
+        require(
+            ICLPool(_clPool).token1() == _oethbAddress,
+            "Only OETHb supported as token1"
+        );
+    }
+
+    /**
+     * @notice initialize function, to set up initial internal state
+     * @param _rewardTokenAddresses Address of reward token for platform
+     */
+    function initialize(
+        address[] memory _rewardTokenAddresses,
+        address[] memory,
+        address[] memory
```

Comment:
These two params can be removed. This is called only from the deployment file and it's not a overriden function either. So, no reason to keep it around

Reply:
Great point, done here: https://github.com/OriginProtocol/origin-dollar/pull/2184/commits/6dcdc87c67278a74fd3d40b31825baf2a830e97e